### DISCUSSION TITLE
Question about the Intestinal permeability parameters & their role to intestinal metabolism

### DISCUSSION THREAD

**Discussion by AyaSaleh90** [Open-Systems-Pharmacology/Forum#1780] [Questions & Problems]
Hi everyone,

I have a question regarding the intestinal permeability parameters. I understand that the specific **intestinal permeability (transcellular)** is calculated based on the lipophilicity and MWeff of the compound. However, there are two additional parameters in the advanced settings: **mucosa P(intracellular -> interstitial)** and **mucosa P(interstitial -> intracellular)**.

Could someone explain the difference between these parameters? Specifically, how do these parameters relate to accounting for intestinal metabolism in the model?

Thanks in advance, 
Aya

**Comment by DeniTue** [comment]
Dear Aya,

The specific intestinal permeability (transcellular) describes compound permeability from the gut lumen into the mucosa cells (apical). The parameters P(interstitial -> intracellular) und P(intracellular -> interstitial) describe the permeability of the compound between mucosa cells and interstitial space (basolateral). Both are calculated from organ permeability multiplied by a scale factor that takes into account the intestinal permeability (transcellular) applied in the model (e.g., manually entered) divided by the "default" intestinal permeability (transcellular) calculated from lipophilicity and MWeff. To my knowledge, the permeability parameters do not take intestinal metabolism directly into account.

You can find detailed information about the calculation of different permeabilites in the documentation: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#absorption

Best regards,
Denise

**Comment by AyaSaleh90** [comment]
Dear Denise,
Thanks for your detailed explanation and the helpful documentation link.

From the documentation, I understood that if the specific intestinal permeability (transcellular) is modified in the simulation, the permeability between the intracellular and interstitial space within the mucosa (P (intracellular -> interstitial)) will also be automatically scaled by the same factor. This adjustment prevents a disproportion between the permeability of the apical and basolateral sides of the enterocytes, avoiding drug accumulation in the enterocytes.

To clarify my understanding: If I aim to increase the **contact time or residence time** between the drug and the intestinal enzymes (such as CYP3A4), which are localized intracellularly (in the apical part of the intestine), the approach should be:
1. **Increase** the intestinal permeability (transcellular) to facilitate the faster entry of the drug into the enterocytes.
2. And/OR **decrease** the mucosa P(intracellular -> interstitial) to slow down the absorption of the drug into blood stream.

Does this interpretation correctly align with the mechanism for optimising contact time with intestinal enzymes?

Thank you again for your assistance.

Kind regards,
Aya

**Comment by DeniTue** [comment]
Dear Aya,

For me this approach sounds plausible. Maybe someone else can comment who has experience in changing gut permeability with the aim to modify intestinal metabolism.

Best regards,
Denise

**Comment by tobiasK2001** [comment]
Dear Aya,

that is also my understanding. If parameters P(interstitial -> intracellular) and P(intracellular -> interstitial) are decreased drug absorption is slowed down and at the same time more drug is metabolized by the intestinal enzymes (e.g. CYP3A4) as the residual tim inside the cell and thus the availability for metabolism is increased. That leads to a higher fraction metabolised intestinal and a lower F.
Hope this might help you.

Best, Tobias

**Comment by AyaSaleh90** [comment]
Dear @DeniTue  & @tobiasK2001,
Thanks a lot for the explanation and yes it is more clear now :) 

Kind regards, 
Aya
